tilequeue/store.py

```diff
@@ -364,6 +364,7 @@ def make_store(yml, credentials={}):
 
     elif store_type == 's3':
         bucket = yml.get('name')
+        path = yml.get('path')
```

Comment:
Ooops! Thanks for fixing this. I'm slightly surprised that `flake8` didn't catch it.